src/fixed-footer-layout.tsx

```diff
@@ -35,6 +34,7 @@ const useStyles = createUseStyles((theme) => ({
     containerSmall: {
         paddingBottom: ({height}) => height,
         backgroundColor: ({containerBgColor}) => containerBgColor || theme.colors.background,
+        minHeight: ({topDistance}) => `calc(100vh - ${topDistance}px)`,
```

Comment:
I think that with this change, I can get rid of this div  I had to mainain in order to fill the background color in feedbackScreen.
https://github.com/Telefonica/mistica-web/pull/343/commits/782f7c84726f0947b2e0c4fe5b4badd83dca13d9#diff-a73f3b991d7fd52028c47266987d26b2caef049af23f23ec6adffd8c12d9231aR276
We have to remember to check it after mergin any of the two PRs.

Reply:
This could be not correct :/ At least not with some usages I'm seeing in webapp...

Reply:
I will keep the div then